src/components/MerchantsPage/styles.module.scss

```diff
@@ -140,6 +141,58 @@
   margin: 0 auto;
 }
 
+.bannerContainer {
+  display: grid;
+  grid-template-rows: 1fr;
+  margin-bottom: 150px;
+  max-height: 240px;
+
+  @include for-phone-only {
+    max-height: 400px;
+  }
+}
+
+.bannerBackground {
+  grid-row: 1;
+  grid-column: 1;
+  background-color: #a8192e;
+  height: 240px;
+  width: 100vw;
+  position: relative;
+  left: 50%;
+  right: 50%;
+  margin-top: 100px;
+  margin-left: -50vw;
```

Comment:
This isn’t ideal

Reply:
ahh yeah agree. I copied this from the implementation of the NYC map. Do you know any other ways to achieve the full-screen width effect?